src/utils/mouseOver.ts

```diff
@@ -36,18 +36,22 @@ export const onMouseOver = (props: IProps) => (d: IGroupDataItem | any, i: numbe
   } = props;
   const ix = bins.findIndex((b) => b === d.label);
   // Initial color
-  let color = Color(colors(d.colorRef || String(i)));
+  // let color = Color(colors(d.colorRef || String(i)));
   if (hover) {
+    // @TODO need to investigate an alternative way of doing this with changing counts
```

Comment:
Looking at this, are we losing colours from histograms?

Reply:
Nope this was just applying a colour variant to the bars on hover and restoring it on mouse exit, but this was causing issues with the colour scheme on the overlaid charts. I've simplified it to instead use a short transition and opacity change to help indicate the active bar